modules/dop/services/autotest/global_config.go

```diff
@@ -231,22 +237,22 @@ func (svc *Service) createOrUpdatePipelineCmsGlobalConfigs(cfg *apistructs.AutoT
 		if err != nil {
 			return fmt.Errorf("invalid apiConfig, err: %v", err)
 		}
-		kvs[CmsCfgKeyAPIGlobalConfig] = apistructs.PipelineCmsConfigValue{
+		kvs[CmsCfgKeyAPIGlobalConfig] = &cmspb.PipelineCmsConfigValue{
 			Value:       string(b),
 			EncryptInDB: false,
-			Type:        apistructs.PipelineCmsConfigTypeKV,
-			Operations:  &apistructs.PipelineCmsConfigDefaultOperationsForKV,
+			Type:        cms.ConfigTypeKV,
+			Operations:  &cms.DefaultOperationsForKV,
 			Comment:     "auto test api global config",
 			From:        apistructs.PipelineSourceAutoTest.String(),
 		}
 
 		// 转成 config.autotest.xx 语法由 pipeline 渲染
 		for _, item := range cfg.APIConfig.Global {
-			kvs[apistructs.PipelineSourceAutoTest.String()+"."+item.Name] = apistructs.PipelineCmsConfigValue{
+			kvs[apistructs.PipelineSourceAutoTest.String()+"."+item.Name] = &cmspb.PipelineCmsConfigValue{
```

Comment:
is this key pattern any description? you should comment it

Reply:
It's dop side code. I won't care about inner logic in the PR.